models/visualization/actions_count.sql

```diff
@@ -53,7 +53,7 @@ FROM (
 	pct_of_organic_sessions_30d
 	FROM {{ ref('actions_hierarchy') }}
 	WHERE content_action != '' AND content_action is not null	
-	AND content_action not like 'rising%'
+	AND content_trajectory not like 'rising%'
```

Comment:
@Mranfevb92 shouldn't this filter be `content_action != ''`?  Goal is to filter for non-blank content actions (the previous filter was due to 'Rising content' not being a true action.

Reply:
Oh ok, though that the purpose was to keep only the pages with potential content problems, assuming that "rising content" wouldn't have any problem. 

I'll change this up.